libnd4j/include/ops/declarable/generic/transforms/standardize.cpp

```diff
@@ -0,0 +1,133 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2019 Skymind, Inc.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+//
+// @author Paul Dubs
+//
+
+#include <op_boilerplate.h>
+#if NOT_EXCLUDED(OP_standardize)
+
+#include <ops/declarable/CustomOperations.h>
+#include <ops/declarable/helpers/reverse.h>
+
+
+namespace nd4j {
+namespace ops  {
+
+    CONFIGURABLE_OP_IMPL(standardize, 1, 1, true, 0, -2) {
+        
+        auto input = INPUT_VARIABLE(0);
+        auto output = OUTPUT_VARIABLE(0);
+        
+        std::vector<int> axis;        
+
+        if (block.width() > 1)            
+            axis = INPUT_VARIABLE(1)->template asVectorT<int>();
+        else if (block.numI() > 0) 
+            axis = *block.getIArguments();        
+
+        if(axis.empty()) {      // do not perform standardization
+            output->assign(input);
```

Comment:
Is "no axis" a legitimate use case? More likely to be a bug than "I want to implement identity op via standardize".
IMO exception would be better.